splunk/src/main/java/com/splunk/HttpService.java

```diff
@@ -545,7 +547,22 @@ public void checkServerTrusted(X509Certificate[] certs, String authType) {
             }
             SSLContext context = SSLContext.getInstance(contextStr);
 
+            TrustManager[] trustAll = new TrustManager[]{
+                    new X509TrustManager() {
+                        public X509Certificate[] getAcceptedIssuers() {
+                            return null;
+                        }
+
+                        public void checkClientTrusted(X509Certificate[] certs, String authType) {
+                        }
+
+                        public void checkServerTrusted(X509Certificate[] certs, String authType) {
+                        }
+                    }
+            };
             context.init(null, trustAll, new java.security.SecureRandom());
+
+
             return new SplunkHttpsSocketFactory(context.getSocketFactory());

```

Comment:
I suggest removing this `SplunkHttpsSocketFactory` class entirely, I don't see any aspect that we are customizing where we wouldn't use the standard `SSLSocketFactory`.

Reply:
Okay